docs/specification.md

```diff
@@ -197,14 +190,13 @@ These values are:
 
 This chapter provides a high-level overview of the contract's logic.
 
-- The contract maintains a ledger of address and its balance (frozen tokens and optionally others).
+- The contract maintains a map of addresses and its balance (frozen tokens).
 - The contract maintains the address of an a FA2 contract and a `token_id`,
   to use in the governance process.
-- The contract manages two special roles, the `admin` and `guardian`.
+- The contract manages three special roles, the `admin`, `guardian`, and `delegate`.
 - The contract stores a list of proposals that can be in one of the states:
-  "ongoing", "rejected", "accepted" or "expired".
-- The contract forbids transferring XTZ to the contract,
-  because they might be locked forever.
+  "pending", "ongoing", and "expired".
```

Comment:
AFAIK the proposal states have not changed, what is `pending`?

Reply:
Seems like the previous proposal state are out dated. I'm using `pending` here to refer to "not yet voting period", probably have to reword this, hope to get your suggestion as well.

Reply:
I don't think they were really outdated, from their creation onward proposals go through these states:
1. "ongoing" when they have been proposed but can't be voted on yet (I guess we can rename it to "proposed")
2. "ongoing" when it's getting voted on
3. "ongoing" if it's no longer time to vote on it, but it hasn't been flushed yet (I guess this can be called "pending flush")
3. "accepted" or "rejected" when they are flushed, depending on the result
4. "expired" if they haven't been flushed in time

If you agree I think we can make this more clear here.

Reply:
I've updated it as the follow:
```
- The contract stores a list of proposals that can be in one of the states:
  "proposed", "ongoing", "pending flush", "accepted, "rejected", or "expired".
```

Let me know if this is good enough. I also rebased and squashed the commits now.

Reply:
Yep, sounds good to me, except that you forgot a double quote after `accepted`. it should be:
```
"proposed", "ongoing", "pending flush", "accepted", "rejected", or "expired".
```

The rebase also looks good to me :+1: